spec/forms/registration_form_spec.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe RegistrationForm do
+  describe '#save' do
+    context 'given valid user input' do
+      it 'returns true' do
+        form = described_class.new
+
+        expect(form.save(valid_signup_params)).to be(true)
```

Comment:
In general, we prefer explicit tests so that we do not need to open/close multiple files when reading a test.

This means we would copy this inside every single test, and adjust the values based on the context of the test:

```
valid_signup_params = Fabricate(:application,
                                email: "test+#{Time.now.to_i}@nimblehq.co",
                                password: '123456',
                                password_confirmation: '123456',
                                client_id: application.uid)
```

It is related to this section: https://nimblehq.co/compass/development/code-conventions/ruby/rspec/#fixtures-isolation

Of course, we still need helper methods for the most common actions like `login_as user` for requests tests ;-)